packages/admin/admin-date-time/src/datePicker/DatePicker.tsx

```diff
@@ -21,20 +30,23 @@ export interface DatePickerProps extends Omit<InputWithPopperProps, "children" |
     maxDate?: Date;
     minDate?: Date;
     slotProps?: SlotProps;
+    startAdornment?: string | IconName | IconObject | ImportReference;
+    endAdornment?: string | IconName | IconObject | ImportReference;
```

Comment:
you are bringing generator config like code into the component :) I guess that should be a RectNode or so